framework/meta/src/template/template_adjuster.rs

```diff
@@ -28,8 +29,9 @@ impl TemplateAdjuster {
         let cargo_toml_path = self.target.contract_dir().join(ROOT_CARGO_TOML);
         let mut toml = CargoTomlContents::load_from_file(&cargo_toml_path);
 
+        let non_framework_crate_names = toml.local_dependency_paths(FRAMEWORK_CRATE_NAMES);
```

Comment:
Create a method that deletes framework paths directly in the toml object.

Also note that local_dependency_paths does not handle dev-dependencies, so make sure to handle both.